pkg/controller/machinedeployment/machinedeployment_controller_test.go

```diff
@@ -205,3 +226,65 @@ func intstrPtr(i int32) *intstr.IntOrString {
 	intstr := intstr.FromInt(int(i))
 	return &intstr
 }
+
+func expectReconcile(t *testing.T, requests chan reconcile.Request, errors chan error) {
+	t.Helper()
+
+	{
+		ctx, cancel := context.WithTimeout(context.TODO(), timeout)
+		defer cancel()
+
+	LOOP0:
+		for range time.Tick(pollingInterval) {
+			select {
+			case recv := <-requests:
+				if recv == expectedRequest {
```

Comment:
where is `expectedRequest` defined?

Reply:
It was already defined at the top of the file.